src/main/java/com/threeamigos/music/yetanotherspotify/service/ArtistService.java

```diff
@@ -0,0 +1,36 @@
+package com.threeamigos.music.yetanotherspotify.service;
+
+import com.threeamigos.music.yetanotherspotify.model.Artist;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+public class ArtistService {
+
+//    TODO: Transfer Service to external module
+    private List<Artist> mockedArtistsList = new ArrayList<>(List.of(
+            new Artist(1, 1, "Metallica"),
+            new Artist(2,2, "Black Sabbath"),
+            new Artist(3,3,"Kyuss"),
+            new Artist(4,4,"Led Zeppelin"),
+            new Artist(5,5,"Tool")
+    ));
+
+    public List<Artist> getArtists(String artistName) {
+        List<Artist> listWithArtistsToServe = new ArrayList<>();
+        for (Artist artist: mockedArtistsList) {
+            if (artist.getArtistName().equals(artistName)) {
+                log.info("Artist with artistName: " + artistName + " found.");
+                listWithArtistsToServe.add(artist);
+                return listWithArtistsToServe;
+            }
+        }
+        log.info("Artist with artistName: " + artistName + " was not found.");
+        return null;
```

Comment:
throw custom exception instead of returning null.

Reply:
Resolved. Now if an artist does not exist, an exception is thrown.